README.md

```diff
@@ -1,9 +1,79 @@
+
+Learn more about deploying and using Charmed Kubeflow at [https://charmed-kubeflow.io](https://charmed-kubeflow.io).
+
+<br/>
+
+### Key features
+* Centralised, browser-based data science workspaces: **familiar experience**
+* Multi user: **one environment for your whole data science team**
+* NVIDIA GPU support: **accelerate deep learning model training**
+* Apache Spark integration: **empower big data driven model training**
+* Ideation to production: **automate model training & deployment**
+* AutoML: **Hyperparameter tuning, architecture search**
+* Composable: **edge deployment configurations available**
+
+<br/>
+
+### What’s included in Charmed Kubeflow 1.3
+* LDAP Authentication
+* Jupyter Notebooks
+* Work with Python and R
+* TensorFlow, Pytorch, MXNet, XGBoost
+* TFServing, Seldon-Core
+* Katib (autoML)
+* Apache Spark
+* Argo Workflows
+* Kubeflow Pipelines
+
+<br/>
+
+### Why engineers and data scientists choose Charmed Kubeflow
+* Maintenance: Charmed Kubeflow offers up to two years of maintenance on select releases
+* Optional 24/7 support available, [contact us here](https://charmed-kubeflow.io/contact-us) for more information
+* Optional dedicated fully managed service available, [contact us here](https://charmed-kubeflow.io/contact-us) for more information or [learn more about Canonical’s Managed Apps service](https://ubuntu.com/managed/apps).
+* Portability: Charmed Kubeflow can be deployed on any conformant Kubernetes, on any cloud or on-premise
+
+<br/>
 
-### Overview
 This repository hosts the Kubernetes Python Operator for Seldon Core
 (see [CharmHub](https://charmhub.io/?q=seldon-core)).
 
-The Seldon Core Operator is a Python script that wraps the latest released Seldon Core manifest,
-providing lifecycle management and handling events (install, upgrade, integrate, remove).
+The Seldon Core Operator is a Python script that wraps the latest released Seldon Core manifest, providing lifecycle management and handling events (install, upgrade, integrate, remove).
+
+<br/>
+
+### Documentation
+Please see the [official docs site](https://charmed-kubeflow.io/docs) for complete documentation of the Charmed Kubeflow distribution.
+
+### Bugs and feature requests
+If you find a bug in our operator or want to request a specific feature, please file a bug here: 
+[https://github.com/canonical/seldon-core-operator/issues](https://github.com/canonical/seldon-core-operator/issues)
+
+<br/>
+
+### License
+Charmed Kubeflow is free software, distributed under the [Apache Software License, version 2](https://github.com/canonical/seldon-core-operator/blob/master/LICENSE).
+
+<br/>
+
+### Contributing
+Canonical welcomes contributions to Charmed Kubeflow. Please check out our contributing guide if you're interested in contributing to the distribution.
```

Comment:
It would be nice to have a `CONTRIBUTING.md` with these guidelines, but since we don't have any, I would add a bit more details in this section or create the md file.

Reply:
Alternatively link to the Canonical contributor agreement  - https://ubuntu.com/legal/contributors

Reply:
```suggestion
Canonical welcomes contributions to Charmed Kubeflow. Please check out our [contributor agreement](https://ubuntu.com/legal/contributors) if you're interested in contributing to the distribution.
```